### PR TITLE
PHOENIX-6715 Update Omid to 1.1.0

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

**Comment by virajjasani** [comment]
@stoty you have this tested against local m2 cache of omid 1.1.0?

**Comment by stoty** [comment]
Yes, I have, at the time.
But there were changes in Omid since then, and we're gonna have to test again, once the Omid HBase2 API migration has landed in Omid.

File: pom.xml [hunk 1]
```diff
@@ -121,7 +121,7 @@
     <collections.version>3.2.2</collections.version>
     <jodatime.version>2.10.5</jodatime.version>
     <joni.version>2.1.31</joni.version>
-    <omid.version>1.0.2</omid.version>
+    <omid.version>1.1.0-SNAPSHOT</omid.version>
```
Comment (richardantal) [inline pom.xml:124]: I think this version should be updated to 1.1.0 after we have the omid released.

Comment (stoty) [reply pom.xml:124]: That's correct, Richard.

**Comment by stoty** [comment]
The failing tests are known flakeys.

**Comment by virajjasani** [comment]
> The failing tests are known flakeys.

Correct, I just saw that the latest build test failures are limited to that build only. The previous build had the passing.

**Review by virajjasani — APPROVED** [review]
+1

**Pull request closed without merging**